### DISCUSSION TITLE
When running Umami on Supabase, should we enable Row Level Security? If so, should this be in the docs?

### DISCUSSION THREAD

**Discussion by jparismorgan** [umami-software/umami#2418] [General]
After following the tutorial to us Umami on Supabase (https://umami.is/docs/running-on-supabase) you will get this warning on each of your Supabase tables:

<img width="1196" alt="Screenshot 2023-12-04 at 12 26 11 PM" src="https://github.com/umami-software/umami/assets/1396242/4e6e8462-58c0-4d2f-8d5e-cc672c575023">

Is this okay to ignore? If not, should the docs include a step to enable this for each table?

**Comment by GregoryZeng** [comment]
Since you have already provided password in the postgres connection URL, you should turn on RLS to prevent anonymous access.

**Reply by jparismorgan** [reply]
Yes thank you, should have updated here, I turned on RLS and it worked fine. Would be great to add this to the docs so people don't forget to do this.

**Reply by lezan** [reply]
Do we need to turn on RLS for all the tables?
Then, any suggestions on the policies to use?

**Reply by pt232** [reply]
I'd also like to know what the RLS policy should look like
